githubissues/main_test.go

```diff
@@ -161,3 +161,13 @@ func TestConfigs(t *testing.T) {
 		})
 	}
 }
+
+func TestGetGithubRepo(t *testing.T) {
```

Comment:
Nit: Can we add a test where this is not set?

Reply:
Sure. Added!